smart-contracts/assembly/contracts/dns/dns.ts

```diff
@@ -190,6 +190,57 @@ export function owner(binaryArgs: StaticArray<u8>): StaticArray<u8> {
   return [];
 }
 
+/**
+ * Checks if the given address is the owner of the website.
+ *
+ * @param websiteName - The name of the website.
+ * @param potentialOwner - The address of the potential owner.
+ * @returns `true` if the potential owner is the actual owner of the website, `false` otherwise.
+ */
+export function isOwnerOfWebsite(
+  websiteName: string,
+  potentialOwner: Address,
```

Comment:
I think you can use string type for Address. it will make your code a bit simple..